bin/StringsScript/README.md

```diff
@@ -0,0 +1,14 @@
+# StringsScript
+
+## How to contribute to this script:
+
+Most of the fuctionality lives in StringsScriptCore. Any new functionality should also be added to this framework and tested.
+
+## Good things to know:
+
+* You can build this script from the command line by navigating to the root /StringsScript folder and running `swift build`
+	* you can run the debug binary from the command line using `./.build/debug/StringsScript` (from the root StringsScript folder)
```

Comment:
Which arguments must be passed to the command?
```
$ ./.build/debug/StringsScript
❌ Error: Insufficient arguments
```

Reply:
`./Library/Strings.swift` and `./Kickstarter-iOS/Locales`. These are the paths where the files will be created. I updated the README.